libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -764,14 +765,25 @@ export class TurnContext {
      * Gets the locale stored in the turnState.
      */
     public get locale(): string {
-        return this._turnState.get(this._turnLocale);
+        const turnObj = this._turnState.get(this._turn);
```

Comment:
I don't think this works because the turn state key is [a Symbol](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-dialogs/src/memory/scopes/turnMemoryScope.ts#L15). What am I missing?

Reply:
Yes, you'are right, we may need to make `TURN_STATE` into string as well.

Reply:
Agree. a string key would make this work.

Reply:
Is there any issue with making that change? I don't think it would be considered breaking since the `Symbol` isn't exported currently.

Reply:
changed TURN_STATE into string